ans_task.yml

```diff
@@ -0,0 +1,105 @@
+---
+- name: Update packages and install Apache2
+  hosts: all
+  become: yes
+
+  tasks:
+    - name: Install Apache2
+      apt:
+        name: apache2
+        state: present
+        update_cache: yes
+- name: Create directory and index.html
+  hosts: localhost
+  become: yes
+
+  tasks:
+    - name: Create directory /var/www/abc
```

Comment:
pass abc website value as a variable so that users can deploy the website on their own preferred name